api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -303,5 +307,7 @@ public void abortGenomicExtractionJob(DbWorkspace dbWorkspace, String jobId) thr
             cohortExtractionConfig.operationalTerraWorkspaceNamespace,
             cohortExtractionConfig.operationalTerraWorkspaceName,
             dbSubmission.get().getSubmissionId());
+
+    wgsExtractCromwellSubmissionDao.save(dbSubmission.get());
```

Comment:
we don't need this save call anymore since there are no updates to the submission